### PR TITLE
home-assistant: 0.109.6 → 0.110.1

### PR DESCRIPTION
**Pull request by mweinelt**
###### Motivation for this change

New upstream release.

https://www.home-assistant.io/blog/2020/05/20/release-110/

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

@Mic92

### CODE REVIEW

**Comment by mweinelt** [comment]
The tests are currently failing

```
hass: must succeed: curl --fail -H 'x-ha-access: some_secret' http://localhost:8123/api/
hass # [    9.852476] hass[750]: 2020-05-20 22:00:42 WARNING (MainThread) [homeassistant.components.http.ban] Login attempt or request with invalid authentication from 127.0.0.1
hass # curl: (22) The requested URL returned error: 401 Unauthorized
hass: output:
Test "Check that Home Assistant's web interface and API can be reached" failed with error: "command `curl --fail -H 'x-ha-access: some_secret' http://localhost:8123/api/` failed (exit code 22)"
error: command `curl --fail -H 'x-ha-access: some_secret' http://localhost:8123/api/` failed (exit code 22)
```

**Comment by mweinelt** [comment]
The tests were already broken with 0.109.6, so the breakage wasn't introduced with this pull.

File: pkgs/servers/home-assistant/frontend.nix [hunk 1]
```diff
@@ -4,11 +4,11 @@ buildPythonPackage rec {
   # the frontend version corresponding to a specific home-assistant version can be found here
   # https://github.com/home-assistant/home-assistant/blob/master/homeassistant/components/frontend/manifest.json
   pname = "home-assistant-frontend";
-  version = "20200505.0";
+  version = "20200519.1";
```
Comment (Mic92) [inline pkgs/servers/home-assistant/frontend.nix:7]:
```suggestion
  version = "20200519.2";
```

File: pkgs/servers/home-assistant/frontend.nix [hunk 2]
```diff
@@ -4,11 +4,11 @@ buildPythonPackage rec {
...
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "0h19ddgnws5l49fwr4b0j99j4xkhpxkwdsmq71q36akd4s5lj0ll";
+    sha256 = "149v56q5anzdfxf0dw1h39vdmcigx732a7abqjfb0xny5484iq8w";
```
Comment (Mic92) [inline pkgs/servers/home-assistant/frontend.nix:11]:
```suggestion
    sha256 = "0ygs3v1in3mcpnfhrqk44xsl710hrm6k5zsjnaskqicfx4hj4hr2";
```

**Review by Mic92 — CHANGES_REQUESTED** [review]
Except for this the frontend-bump this looks good to me.

File: pkgs/servers/home-assistant/frontend.nix (hunk 1 above)
Comment (mweinelt) [reply pkgs/servers/home-assistant/frontend.nix:7]: https://github.com/home-assistant/core/blob/0.110.1/homeassistant/components/frontend/manifest.json#L5

Hm?

**Comment by Mic92** [comment]
The frontend module already has the right version.

**Pull request merged**